### ISSUE TITLE
WiFi / mwifiex_pcie / Surface Pro 3 rather slow

### ISSUE THREAD

**Issue by dynek** [linux-surface/linux-surface#1600]
[Description of the bug or feature]

### Environment

- Hardware model: Surface Pro 3 - 128Gb
- Kernel version: `Linux surface 6.1.0-26-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.1.112-1 (2024-09-30) x86_64 GNU/Linux` (not running the kernel from this repo (yet, as everything seems to work with the standard kernel).
- Distribution: Debian

<details>
Hey everyone,

Wifi on the device is working quite well, but it's kind of slow, and dmesg is "flooded" with `mwifiex_pcie 0000:01:00.0: wrong rx packet offset: len=106, rx_pkt_off=90`.

Any way known way to fix this ?

Thank you

To give a bit more insight:

<b>Pinging gateway from Surface</b>
![image](https://github.com/user-attachments/assets/84a2c712-596b-4415-aeec-f0cea4b2c3ac)

<b>Pinging Surface from another device in the same subnet</b>
![image](https://github.com/user-attachments/assets/f048b1e3-045b-4e2b-a8d8-7c8d4bdad5d7)

<b>WiFi information from router (Mikrotik)</b>
![image](https://github.com/user-attachments/assets/4a7a6470-a221-4164-b0c1-51bac7a6a93d)

<b>WiFI information from Surface</b>
![image](https://github.com/user-attachments/assets/f7048351-9a38-4b94-8bff-b6282ff9a254)

<summary>`dmesg` output</summary>

```
[    0.000000] microcode: microcode updated early to revision 0x26, date = 2019-11-12
[    0.000000] Linux version 6.1.0-26-amd64 (debian-kernel@lists.debian.org) (gcc-12 (Debian 12.2.0-14) 12.2.0, GNU ld (GNU Binutils for Debian) 2.40) #1 SMP PREEMPT_DYNAMIC Debian 6.1.112-1 (2024-09-30)
[    0.000000] Command line: BOOT_IMAGE=/boot/vmlinuz-6.1.0-26-amd64 root=UUID=6a39fd43-22f9-4292-bce7-00fdce3a0e55 ro apparmor=0 selinux=0 mitigations=off intel_iommu=on,igfx_off
[    0.000000] BIOS-provided physical RAM map:
[    0.000000] BIOS-e820: [mem 0x0000000000000000-0x0000000000057fff] usable
[    0.000000] BIOS-e820: [mem 0x0000000000058000-0x0000000000058fff] reserved
[    0.000000] BIOS-e820: [mem 0x0000000000059000-0x000000000009dfff] usable
[    0.000000] BIOS-e820: [mem 0x000000000009e000-0x000000000009ffff] reserved
[    0.000000] BIOS-e820: [mem 0x0000000000100000-0x00000000aa376fff] usable
[    0.000000] BIOS-e820: [mem 0x00000000aa377000-0x00000000aa3f3fff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000aa3f4000-0x00000000aa40afff] ACPI data
[    0.000000] BIOS-e820: [mem 0x00000000aa40b000-0x00000000abb54fff] ACPI NVS
[    0.000000] BIOS-e820: [mem 0x00000000abb55000-0x00000000abeb1fff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000abeb2000-0x00000000abffefff] type 20
[    0.000000] BIOS-e820: [mem 0x00000000abfff000-0x00000000abffffff] usable
[    0.000000] BIOS-e820: [mem 0x00000000e0000000-0x00000000efffffff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000fec00000-0x00000000fec00fff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000fed00000-0x00000000fed03fff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000fed1c000-0x00000000fed1ffff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000fee00000-0x00000000feefffff] reserved
[    0.000000] BIOS-e820: [mem 0x00000000ff000000-0x00000000ffffffff] reserved
[    0.000000] BIOS-e820: [mem 0x0000000100000000-0x000000014fdfffff] usable
[    0.000000] NX (Execute Disable) protection: active
[    0.000000] efi: EFI v2.31 by American Megatrends
[    0.000000] efi: ESRT=0xabeb0c18 ACPI 2.0=0xaa3f8000 ACPI=0xaa3f8000 SMBIOS=0xabeaf018 MOKvar=0xabbfe000 
[    0.000000] secureboot: Secure boot disabled
[    0.000000] SMBIOS 2.8 present.
[    0.000000] DMI: Microsoft Corporation Surface Pro 3/Surface Pro 3, BIOS 3.11.2650 04/30/2019
[    0.000000] tsc: Fast TSC calibration using PIT
[    0.000000] tsc: Detected 2494.354 MHz processor
[    0.000074] e820: update [mem 0x00000000-0x00000fff] usable ==> reserved
[    0.000078] e820: remove [mem 0x000a0000-0x000fffff] usable
[    0.000090] last_pfn = 0x14fe00 max_arch_pfn = 0x400000000
[    0.000225] x86/PAT: Configuration [0-7]: WB  WC  UC- UC  WB  WP  UC- WT  
[    0.000854] last_pfn = 0xac000 max_arch_pfn = 0x400000000
[    0.020817] esrt: Reserving ESRT space from 0x00000000abeb0c18 to 0x00000000abeb0cc8.
[    0.020829] Kernel/User page tables isolation: disabled on command line.
[    0.020831] Using GB pages for direct mapping
[    0.021386] RAMDISK: [mem 0x33353000-0x359a0fff]
[    0.021393] ACPI: Early table checksum verification disabled
[    0.021396] ACPI: RSDP 0x00000000AA3F8000 000024 (v02 OEMC  )
[    0.021402] ACPI: XSDT 0x00000000AA3F80A0 0000BC (v01 OEMC   O E M C  00000300 AMI  00010013)
[    0.021409] ACPI: FACP 0x00000000AA405328 00010C (v05 OEMC   O E M C  00000300 AMI  00010013)
[    0.021416] ACPI: DSDT 0x00000000AA3F81E8 00D13B (v02 OEMC   O E M C  00000300 INTL 20120913)
[    0.021420] ACPI: FACS 0x00000000ABB4E080 000040
[    0.021424] ACPI: APIC 0x00000000AA405438 000072 (v03 OEMC   O E M C  00000300 AMI  00010013)
[    0.021428] ACPI: FPDT 0x00000000AA4054B0 000044 (v01 OEMC   O E M C  00000300 AMI  00010013)
[    0.021433] ACPI: SSDT 0x00000000AA4054F8 00036D (v01 SataRe SataTabl 00001000 INTL 20120913)
[    0.021437] ACPI: HPET 0x00000000AA405868 000038 (v01 OEMC   O E M C  00000300 AMI. 00000005)
[    0.021441] ACPI: MSDM 0x00000000AA4058A0 000055 (v03 OEMC   O E M C  00000300 AMI  00010013)
[    0.021446] ACPI: WDSA 0x00000000AA4058F8 00018D (v02 OEMC   ACTABL   00000003 INTL 20120913)
[    0.021450] ACPI: LPIT 0x00000000AA405A88 000094 (v01 ALASKA A M I    00000000 AMI. 00000005)
[    0.021454] ACPI: DBGP 0x00000000AA405B20 000034 (v01 OEMC   O E M C  00000300 MSFT 0000005F)
[    0.021458] ACPI: HPET 0x00000000AA405B58 000038 (v01 OEMC   O E M C  00000300 MSFT 0000005F)
[    0.021462] ACPI: MCFG 0x00000000AA405B90 00003C (v01 OEMC   O E M C  00000300 MSFT 0000005F)
[    0.021467] ACPI: SSDT 0x00000000AA405BD0 00047E (v01 OEMC   Ult0Rtd3 00001000 INTL 20120913)
[    0.021471] ACPI: TPM2 0x00000000AA406050 000034 (v03        Tpm2Tabl 00000001 AMI  00000000)
[    0.021475] ACPI: SSDT 0x00000000AA406088 000539 (v01 PmRef  Cpu0Ist  00003000 INTL 20120913)
[    0.021479] ACPI: SSDT 0x00000000AA4065C8 000AD8 (v01 PmRef  CpuPm    00003000 INTL 20120913)
[    0.021484] ACPI: SSDT 0x00000000AA4070A0 003533 (v01 SaSsdt SaSsdt   00003000 INTL 20120913)
[    0.021488] ACPI: UEFI 0x00000000AA40A5D8 000042 (v01 OEMC   O E M C  00000300      00000000)
[    0.021492] ACPI: DMAR 0x00000000AA40A620 0000F0 (v01 INTEL  HSW      00000001 INTL 00000001)
[    0.021497] ACPI: BGRT 0x00000000AA40A710 000038 (v01 OEMC   O E M C  00000300 AMI  00010013)
[    0.021500] ACPI: Reserving FACP table memory at [mem 0xaa405328-0xaa405433]
[    0.021502] ACPI: Reserving DSDT table memory at [mem 0xaa3f81e8-0xaa405322]
[    0.021504] ACPI: Reserving FACS table memory at [mem 0xabb4e080-0xabb4e0bf]
[    0.021505] ACPI: Reserving APIC table memory at [mem 0xaa405438-0xaa4054a9]
[    0.021506] ACPI: Reserving FPDT table memory at [mem 0xaa4054b0-0xaa4054f3]
[    0.021508] ACPI: Reserving SSDT table memory at [mem 0xaa4054f8-0xaa405864]
[    0.021509] ACPI: Reserving HPET table memory at [mem 0xaa405868-0xaa40589f]
[    0.021510] ACPI: Reserving MSDM table memory at [mem 0xaa4058a0-0xaa4058f4]
[    0.021511] ACPI: Reserving WDSA table memory at [mem 0xaa4058f8-0xaa405a84]
[    0.021513] ACPI: Reserving LPIT table memory at [mem 0xaa405a88-0xaa405b1b]
[    0.021514] ACPI: Reserving DBGP table memory at [mem 0xaa405b20-0xaa405b53]
[    0.021515] ACPI: Reserving HPET table memory at [mem 0xaa405b58-0xaa405b8f]
[    0.021516] ACPI: Reserving MCFG table memory at [mem 0xaa405b90-0xaa405bcb]
[    0.021518] ACPI: Reserving SSDT table memory at [mem 0xaa405bd0-0xaa40604d]
[    0.021519] ACPI: Reserving TPM2 table memory at [mem 0xaa406050-0xaa406083]
[    0.021520] ACPI: Reserving SSDT table memory at [mem 0xaa406088-0xaa4065c0]
[    0.021521] ACPI: Reserving SSDT table memory at [mem 0xaa4065c8-0xaa40709f]
[    0.021523] ACPI: Reserving SSDT table memory at [mem 0xaa4070a0-0xaa40a5d2]
[    0.021524] ACPI: Reserving UEFI table memory at [mem 0xaa40a5d8-0xaa40a619]
[    0.021525] ACPI: Reserving DMAR table memory at [mem 0xaa40a620-0xaa40a70f]
[    0.021526] ACPI: Reserving BGRT table memory at [mem 0xaa40a710-0xaa40a747]
[    0.021600] No NUMA configuration found
[    0.021601] Faking a node at [mem 0x0000000000000000-0x000000014fdfffff]
[    0.021617] NODE_DATA(0) allocated [mem 0x14fdd3000-0x14fdfdfff]
[    0.022024] Zone ranges:
[    0.022025]   DMA      [mem 0x0000000000001000-0x0000000000ffffff]
[    0.022028]   DMA32    [mem 0x0000000001000000-0x00000000ffffffff]
[    0.022030]   Normal   [mem 0x0000000100000000-0x000000014fdfffff]
[    0.022032]   Device   empty
[    0.022033] Movable zone start for each node
[    0.022038] Early memory node ranges
[    0.022038]   node   0: [mem 0x0000000000001000-0x0000000000057fff]
[    0.022041]   node   0: [mem 0x0000000000059000-0x000000000009dfff]
[    0.022042]   node   0: [mem 0x0000000000100000-0x00000000aa376fff]
[    0.022044]   node   0: [mem 0x00000000abfff000-0x00000000abffffff]
[    0.022045]   node   0: [mem 0x0000000100000000-0x000000014fdfffff]
[    0.022048] Initmem setup node 0 [mem 0x0000000000001000-0x000000014fdfffff]
[    0.022054] On node 0, zone DMA: 1 pages in unavailable ranges
[    0.022057] On node 0, zone DMA: 1 pages in unavailable ranges
[    0.022108] On node 0, zone DMA: 98 pages in unavailable ranges
[    0.030740] On node 0, zone DMA32: 7304 pages in unavailable ranges
[    0.031423] On node 0, zone Normal: 16384 pages in unavailable ranges
[    0.031434] On node 0, zone Normal: 512 pages in unavailable ranges
[    0.031443] Reserving Intel graphics memory at [mem 0xad200000-0xaf1fffff]
[    0.031616] ACPI: PM-Timer IO Port: 0x1808
[    0.031625] ACPI: LAPIC_NMI (acpi_id[0xff] high edge lint[0x1])
[    0.031638] IOAPIC[0]: apic_id 8, version 32, address 0xfec00000, GSI 0-39
[    0.031642] ACPI: INT_SRC_OVR (bus 0 bus_irq 0 global_irq 2 dfl dfl)
[    0.031645] ACPI: INT_SRC_OVR (bus 0 bus_irq 9 global_irq 9 high level)
[    0.031651] ACPI: Using ACPI (MADT) for SMP configuration information
[    0.031652] ACPI: HPET id: 0x8086a701 base: 0xfed00000
[    0.031663] e820: update [mem 0xa88ad000-0xa88ccfff] usable ==> reserved
[    0.031680] TSC deadline timer available
[    0.031681] smpboot: Allowing 4 CPUs, 0 hotplug CPUs
[    0.031709] PM: hibernation: Registered nosave memory: [mem 0x00000000-0x00000fff]
[    0.031712] PM: hibernation: Registered nosave memory: [mem 0x00058000-0x00058fff]
[    0.031715] PM: hibernation: Registered nosave memory: [mem 0x0009e000-0x0009ffff]
[    0.031716] PM: hibernation: Registered nosave memory: [mem 0x000a0000-0x000fffff]
[    0.031718] PM: hibernation: Registered nosave memory: [mem 0xa88ad000-0xa88ccfff]
[    0.031721] PM: hibernation: Registered nosave memory: [mem 0xaa377000-0xaa3f3fff]
[    0.031722] PM: hibernation: Registered nosave memory: [mem 0xaa3f4000-0xaa40afff]
[    0.031723] PM: hibernation: Registered nosave memory: [mem 0xaa40b000-0xabb54fff]
[    0.031724] PM: hibernation: Registered nosave memory: [mem 0xabb55000-0xabeb1fff]
[    0.031725] PM: hibernation: Registered nosave memory: [mem 0xabeb2000-0xabffefff]
[    0.031728] PM: hibernation: Registered nosave memory: [mem 0xac000000-0xad1fffff]
[    0.031729] PM: hibernation: Registered nosave memory: [mem 0xad200000-0xaf1fffff]
[    0.031730] PM: hibernation: Registered nosave memory: [mem 0xaf200000-0xdfffffff]
[    0.031731] PM: hibernation: Registered nosave memory: [mem 0xe0000000-0xefffffff]
[    0.031732] PM: hibernation: Registered nosave memory: [mem 0xf0000000-0xfebfffff]
[    0.031733] PM: hibernation: Registered nosave memory: [mem 0xfec00000-0xfec00fff]
[    0.031734] PM: hibernation: Registered nosave memory: [mem 0xfec01000-0xfecfffff]
[    0.031735] PM: hibernation: Registered nosave memory: [mem 0xfed00000-0xfed03fff]
[    0.031736] PM: hibernation: Registered nosave memory: [mem 0xfed04000-0xfed1bfff]
[    0.031737] PM: hibernation: Registered nosave memory: [mem 0xfed1c000-0xfed1ffff]
[    0.031738] PM: hibernation: Registered nosave memory: [mem 0xfed20000-0xfedfffff]
[    0.031739] PM: hibernation: Registered nosave memory: [mem 0xfee00000-0xfeefffff]
[    0.031740] PM: hibernation: Registered nosave memory: [mem 0xfef00000-0xfeffffff]
[    0.031741] PM: hibernation: Registered nosave memory: [mem 0xff000000-0xffffffff]
[    0.031744] [mem 0xaf200000-0xdfffffff] available for PCI devices
[    0.031746] Booting paravirtualized kernel on bare hardware
[    0.031749] clocksource: refined-jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 7645519600211568 ns
[    0.038572] setup_percpu: NR_CPUS:8192 nr_cpumask_bits:4 nr_cpu_ids:4 nr_node_ids:1
[    0.038840] percpu: Embedded 61 pages/cpu s212992 r8192 d28672 u524288
[    0.038850] pcpu-alloc: s212992 r8192 d28672 u524288 alloc=1*2097152
[    0.038854] pcpu-alloc: [0] 0 1 2 3 
[    0.038895] Fallback order for Node 0: 0 
[    0.038899] Built 1 zonelists, mobility grouping on.  Total pages: 1008114
[    0.038901] Policy zone: Normal
[    0.038902] Kernel command line: BOOT_IMAGE=/boot/vmlinuz-6.1.0-26-amd64 root=UUID=6a39fd43-22f9-4292-bce7-00fdce3a0e55 ro apparmor=0 selinux=0 mitigations=off intel_iommu=on,igfx_off
[    0.039019] DMAR: IOMMU enabled
[    0.039020] DMAR: Disable GFX device mapping
[    0.039024] Unknown kernel command line parameters "BOOT_IMAGE=/boot/vmlinuz-6.1.0-26-amd64 selinux=0", will be passed to user space.
[    0.039046] random: crng init done
[    0.039335] Dentry cache hash table entries: 524288 (order: 10, 4194304 bytes, linear)
[    0.039482] Inode-cache hash table entries: 262144 (order: 9, 2097152 bytes, linear)
[    0.039548] mem auto-init: stack:all(zero), heap alloc:on, heap free:off
[    0.039557] software IO TLB: area num 4.
[    0.064806] Memory: 2728820K/4097104K available (14342K kernel code, 2335K rwdata, 9076K rodata, 2796K init, 17396K bss, 264104K reserved, 0K cma-reserved)
[    0.065001] SLUB: HWalign=64, Order=0-3, MinObjects=0, CPUs=4, Nodes=1
[    0.065036] ftrace: allocating 40246 entries in 158 pages
[    0.074729] ftrace: allocated 158 pages with 5 groups
[    0.075639] Dynamic Preempt: voluntary
[    0.075677] rcu: Preemptible hierarchical RCU implementation.
[    0.075679] rcu: 	RCU restricting CPUs from NR_CPUS=8192 to nr_cpu_ids=4.
[    0.075680] 	Trampoline variant of Tasks RCU enabled.
[    0.075681] 	Rude variant of Tasks RCU enabled.
[    0.075682] 	Tracing variant of Tasks RCU enabled.
[    0.075683] rcu: RCU calculated value of scheduler-enlistment delay is 25 jiffies.
[    0.075684] rcu: Adjusting geometry for rcu_fanout_leaf=16, nr_cpu_ids=4
[    0.082099] NR_IRQS: 524544, nr_irqs: 728, preallocated irqs: 16
[    0.082300] rcu: srcu_init: Setting srcu_struct sizes based on contention.
[    0.082392] Console: colour dummy device 80x25
[    0.082817] printk: console [tty0] enabled
[    0.082841] ACPI: Core revision 20220331
[    0.082963] hpet: HPET dysfunctional in PC10. Force disabled.
[    0.082967] APIC: Switch to symmetric I/O mode setup
[    0.082972] DMAR: Host address width 39
[    0.082975] DMAR: DRHD base: 0x000000fed90000 flags: 0x0
[    0.082986] DMAR: dmar0: reg_base_addr fed90000 ver 1:0 cap c0000020660462 ecap f0101a
[    0.082993] DMAR: DRHD base: 0x000000fed91000 flags: 0x1
[    0.083000] DMAR: dmar1: reg_base_addr fed91000 ver 1:0 cap d2008020660462 ecap f010da
[    0.083005] DMAR: RMRR base: 0x0000003e2e0000 end: 0x0000003e2fffff
[    0.083009] DMAR: [Firmware Bug]: No firmware reserved region can cover this RMRR [0x000000003e2e0000-0x000000003e2fffff], contact BIOS vendor for fixes
[    0.083015] DMAR: [Firmware Bug]: Your BIOS is broken; bad RMRR [0x000000003e2e0000-0x000000003e2fffff]
[    0.083015] BIOS vendor: American Megatrends Inc.; Ver: 3.11.2650; Product Version: 1
[    0.083024] DMAR: RMRR base: 0x000000ad000000 end: 0x000000af1fffff
[    0.083028] DMAR: [Firmware Bug]: No firmware reserved region can cover this RMRR [0x00000000ad000000-0x00000000af1fffff], contact BIOS vendor for fixes
[    0.083033] DMAR: [Firmware Bug]: Your BIOS is broken; bad RMRR [0x00000000ad000000-0x00000000af1fffff]
[    0.083033] BIOS vendor: American Megatrends Inc.; Ver: 3.11.2650; Product Version: 1
[    0.083041] DMAR: ANDD device: 1 name: \_SB.PCI0.I2C0
[    0.083045] DMAR: ANDD device: 2 name: \_SB.PCI0.I2C1
[    0.083049] DMAR-IR: IOAPIC id 8 under DRHD base  0xfed91000 IOMMU 1
[    0.083054] DMAR-IR: HPET id 0 under DRHD base 0xfed91000
[    0.083057] DMAR-IR: Queued invalidation will be enabled to support x2apic and Intr-remapping.
[    0.083727] DMAR-IR: Enabled IRQ remapping in x2apic mode
[    0.083734] x2apic enabled
[    0.083744] Switched APIC routing to cluster x2apic.
[    0.084187] clocksource: tsc-early: mask: 0xffffffffffffffff max_cycles: 0x23f4641691f, max_idle_ns: 440795264483 ns
[    0.084199] Calibrating delay loop (skipped), value calculated using timer frequency.. 4988.70 BogoMIPS (lpj=9977416)
[    0.084236] CPU0: Thermal monitoring enabled (TM1)
[    0.084276] process: using mwait in idle threads
[    0.084281] Last level iTLB entries: 4KB 1024, 2MB 1024, 4MB 1024
[    0.084285] Last level dTLB entries: 4KB 1024, 2MB 1024, 4MB 1024, 1GB 4
[    0.084292] Spectre V2 : User space: Vulnerable
[    0.084296] Speculative Store Bypass: Vulnerable
[    0.084301] SRBDS: Vulnerable
[    0.084308] x86/fpu: Supporting XSAVE feature 0x001: 'x87 floating point registers'
[    0.084313] x86/fpu: Supporting XSAVE feature 0x002: 'SSE registers'
[    0.084317] x86/fpu: Supporting XSAVE feature 0x004: 'AVX registers'
[    0.084321] x86/fpu: xstate_offset[2]:  576, xstate_sizes[2]:  256
[    0.084325] x86/fpu: Enabled xstate features 0x7, context size is 832 bytes, using 'standard' format.
[    0.088195] Freeing SMP alternatives memory: 36K
[    0.088195] pid_max: default: 32768 minimum: 301
[    0.088195] LSM: Security Framework initializing
[    0.088195] landlock: Up and running.
[    0.088195] Yama: disabled by default; enable with sysctl kernel.yama.*
[    0.088195] SELinux:  Initializing.
[    0.088195] TOMOYO Linux initialized
[    0.088195] LSM support for eBPF active
[    0.088195] Mount-cache hash table entries: 8192 (order: 4, 65536 bytes, linear)
[    0.088195] Mountpoint-cache hash table entries: 8192 (order: 4, 65536 bytes, linear)
[    0.088195] smpboot: CPU0: Intel(R) Core(TM) i5-4300U CPU @ 1.90GHz (family: 0x6, model: 0x45, stepping: 0x1)
[    0.088195] cblist_init_generic: Setting adjustable number of callback queues.
[    0.088195] cblist_init_generic: Setting shift to 2 and lim to 1.
[    0.088195] cblist_init_generic: Setting adjustable number of callback queues.
[    0.088195] cblist_init_generic: Setting shift to 2 and lim to 1.
[    0.088195] cblist_init_generic: Setting adjustable number of callback queues.
[    0.088195] cblist_init_generic: Setting shift to 2 and lim to 1.
[    0.088195] Performance Events: PEBS fmt2+, Haswell events, 16-deep LBR, full-width counters, Intel PMU driver.
[    0.088195] ... version:                3
[    0.088195] ... bit width:              48
[    0.088195] ... generic registers:      4
[    0.088195] ... value mask:             0000ffffffffffff
[    0.088195] ... max period:             00007fffffffffff
[    0.088195] ... fixed-purpose events:   3
[    0.088195] ... event mask:             000000070000000f
[    0.088195] signal: max sigframe size: 1776
[    0.088195] Estimated ratio of average max frequency by base frequency (times 1024): 1064
[    0.088195] rcu: Hierarchical SRCU implementation.
[    0.088195] rcu: 	Max phase no-delay instances is 1000.
[    0.088195] NMI watchdog: Enabled. Permanently consumes one hw-PMU counter.
[    0.088195] smp: Bringing up secondary CPUs ...
[    0.088195] x86: Booting SMP configuration:
[    0.088195] .... node  #0, CPUs:      #1 #2 #3
[    0.089646] smp: Brought up 1 node, 4 CPUs
[    0.089646] smpboot: Max logical packages: 1
[    0.089646] smpboot: Total of 4 processors activated (19954.83 BogoMIPS)
[    0.094400] node 0 deferred pages initialised in 0ms
[    0.094887] devtmpfs: initialized
[    0.094887] x86/mm: Memory block size: 128MB
[    0.096355] ACPI: PM: Registering ACPI NVS region [mem 0xaa40b000-0xabb54fff] (24420352 bytes)
[    0.096755] clocksource: jiffies: mask: 0xffffffff max_cycles: 0xffffffff, max_idle_ns: 7645041785100000 ns
[    0.096769] futex hash table entries: 1024 (order: 4, 65536 bytes, linear)
[    0.096828] pinctrl core: initialized pinctrl subsystem
[    0.097571] NET: Registered PF_NETLINK/PF_ROUTE protocol family
[    0.097751] DMA: preallocated 512 KiB GFP_KERNEL pool for atomic allocations
[    0.097805] DMA: preallocated 512 KiB GFP_KERNEL|GFP_DMA pool for atomic allocations
[    0.097856] DMA: preallocated 512 KiB GFP_KERNEL|GFP_DMA32 pool for atomic allocations
[    0.097870] audit: initializing netlink subsys (disabled)
[    0.097882] audit: type=2000 audit(1730143110.012:1): state=initialized audit_enabled=0 res=1
[    0.097882] thermal_sys: Registered thermal governor 'fair_share'
[    0.097882] thermal_sys: Registered thermal governor 'bang_bang'
[    0.097882] thermal_sys: Registered thermal governor 'step_wise'
[    0.097882] thermal_sys: Registered thermal governor 'user_space'
[    0.097882] thermal_sys: Registered thermal governor 'power_allocator'
[    0.097882] cpuidle: using governor ladder
[    0.097882] cpuidle: using governor menu
[    0.097882] acpiphp: ACPI Hot Plug PCI Controller Driver version: 0.5
[    0.097882] PCI: MMCONFIG for domain 0000 [bus 00-ff] at [mem 0xe0000000-0xefffffff] (base 0xe0000000)
[    0.097882] PCI: MMCONFIG at [mem 0xe0000000-0xefffffff] reserved in E820
[    0.097882] PCI: Using configuration type 1 for base access
[    0.097882] core: PMU erratum BJ122, BV98, HSD29 worked around, HT is on
[    0.097882] ENERGY_PERF_BIAS: Set to 'normal', was 'performance'
[    0.097882] mtrr: your CPUs had inconsistent variable MTRR settings
[    0.097882] mtrr: probably your BIOS does not setup all CPUs.
[    0.097882] mtrr: corrected configuration.
[    0.097882] kprobes: kprobe jump-optimization is enabled. All kprobes are optimized if possible.
[    0.097882] HugeTLB: registered 1.00 GiB page size, pre-allocated 0 pages
[    0.097882] HugeTLB: 16380 KiB vmemmap can be freed for a 1.00 GiB page
[    0.097882] HugeTLB: registered 2.00 MiB page size, pre-allocated 0 pages
[    0.097882] HugeTLB: 28 KiB vmemmap can be freed for a 2.00 MiB page
[    0.100540] ACPI: Added _OSI(Module Device)
[    0.100547] ACPI: Added _OSI(Processor Device)
[    0.100552] ACPI: Added _OSI(3.0 _SCP Extensions)
[    0.100557] ACPI: Added _OSI(Processor Aggregator Device)
[    0.115861] ACPI: 6 ACPI AML tables successfully acquired and loaded
[    0.117656] ACPI: [Firmware Bug]: BIOS _OSI(Linux) query ignored
[    0.118702] ACPI: Dynamic OEM Table Load:
[    0.118718] ACPI: SSDT 0xFFFF9C3C00BBD400 0003D3 (v01 PmRef  Cpu0Cst  00003001 INTL 20120913)
[    0.119898] ACPI: Dynamic OEM Table Load:
[    0.119912] ACPI: SSDT 0xFFFF9C3C01011000 0005AA (v01 PmRef  ApIst    00003000 INTL 20120913)
[    0.121190] ACPI: Dynamic OEM Table Load:
[    0.121202] ACPI: SSDT 0xFFFF9C3C01058800 000119 (v01 PmRef  ApCst    00003000 INTL 20120913)
[    0.123243] ACPI: EC: EC started
[    0.123249] ACPI: EC: interrupt blocked
[    0.127022] ACPI: EC: EC_CMD/EC_SC=0x66, EC_DATA=0x62
[    0.127030] ACPI: \_SB_.PCI0.SBRG.EC0_: Boot DSDT EC used to handle transactions
[    0.127037] ACPI: Interpreter enabled
[    0.127079] ACPI: PM: (supports S0 S4 S5)
[    0.127084] ACPI: Using IOAPIC for interrupt routing
[    0.127132] PCI: Using host bridge windows from ACPI; if necessary, use "pci=nocrs" and report a bug
[    0.127140] PCI: Using E820 reservations for host bridge windows
[    0.127724] ACPI: Enabled 10 GPEs in block 00 to 7F
[    0.129274] ACPI: PM: Power Resource [PRWF]
[    0.132932] ACPI: PM: Power Resource [CAMP]
[    0.153238] ACPI: PM: Power Resource [PAUD]
[    0.155495] ACPI: PM: Power Resource [TPWR]
[    0.221232] ACPI: PCI Root Bridge [PCI0] (domain 0000 [bus 00-fe])
[    0.221244] acpi PNP0A08:00: _OSC: OS supports [ExtendedConfig ASPM ClockPM Segments MSI HPX-Type3]
[    0.221753] acpi PNP0A08:00: _OSC: OS now controls [PCIeHotplug SHPCHotplug PME AER PCIeCapability LTR]
[    0.222349] PCI host bridge to bus 0000:00
[    0.222354] pci_bus 0000:00: root bus resource [io  0x0000-0x0cf7 window]
[    0.222360] pci_bus 0000:00: root bus resource [io  0x0d00-0xffff window]
[    0.222366] pci_bus 0000:00: root bus resource [mem 0x000a0000-0x000fffff window]
[    0.222370] pci_bus 0000:00: root bus resource [mem 0xfed40000-0xfed44fff window]
[    0.222374] pci_bus 0000:00: root bus resource [mem 0xaf200000-0xfeafffff window]
[    0.222379] pci_bus 0000:00: root bus resource [bus 00-fe]
[    0.222401] pci 0000:00:00.0: [8086:0a04] type 00 class 0x060000
[    0.222493] pci 0000:00:02.0: [8086:0a16] type 00 class 0x030000
[    0.222506] pci 0000:00:02.0: reg 0x10: [mem 0xc0000000-0xc03fffff 64bit]
[    0.222515] pci 0000:00:02.0: reg 0x18: [mem 0xb0000000-0xbfffffff 64bit pref]
[    0.222523] pci 0000:00:02.0: reg 0x20: [io  0x3000-0x303f]
[    0.222540] pci 0000:00:02.0: BAR 2: assigned to efifb
[    0.222545] pci 0000:00:02.0: Video device with shadowed ROM at [mem 0x000c0000-0x000dffff]
[    0.222657] pci 0000:00:03.0: [8086:0a0c] type 00 class 0x040300
[    0.222669] pci 0000:00:03.0: reg 0x10: [mem 0xc0614000-0xc0617fff 64bit]
[    0.222794] pci 0000:00:14.0: [8086:9c31] type 00 class 0x0c0330
[    0.222815] pci 0000:00:14.0: reg 0x10: [mem 0xc0600000-0xc060ffff 64bit]
[    0.222878] pci 0000:00:14.0: PME# supported from D3hot D3cold
[    0.223000] pci 0000:00:16.0: [8086:9c3a] type 00 class 0x078000
[    0.223022] pci 0000:00:16.0: reg 0x10: [mem 0xc061c000-0xc061c01f 64bit]
[    0.223096] pci 0000:00:16.0: PME# supported from D0 D3hot D3cold
[    0.223194] pci 0000:00:1b.0: [8086:9c20] type 00 class 0x040300
[    0.223213] pci 0000:00:1b.0: reg 0x10: [mem 0xc0610000-0xc0613fff 64bit]
[    0.223279] pci 0000:00:1b.0: PME# supported from D0 D3hot D3cold
[    0.223385] pci 0000:00:1c.0: [8086:9c14] type 01 class 0x060400
[    0.223482] pci 0000:00:1c.0: PME# supported from D0 D3hot D3cold
[    0.223520] pci 0000:00:1c.0: Enabling MPC IRBNCE
[    0.223524] pci 0000:00:1c.0: Intel PCH root port ACS workaround enabled
[    0.223811] pci 0000:00:1f.0: [8086:9c43] type 00 class 0x060100
[    0.224026] pci 0000:00:1f.2: [8086:9c03] type 00 class 0x010601
[    0.224042] pci 0000:00:1f.2: reg 0x10: [io  0x30b0-0x30b7]
[    0.224053] pci 0000:00:1f.2: reg 0x14: [io  0x30a0-0x30a3]
[    0.224062] pci 0000:00:1f.2: reg 0x18: [io  0x3090-0x3097]
[    0.224070] pci 0000:00:1f.2: reg 0x1c: [io  0x3080-0x3083]
[    0.224079] pci 0000:00:1f.2: reg 0x20: [io  0x3060-0x307f]
[    0.224088] pci 0000:00:1f.2: reg 0x24: [mem 0xc061a000-0xc061a7ff]
[    0.224128] pci 0000:00:1f.2: PME# supported from D3hot
[    0.224215] pci 0000:00:1f.3: [8086:9c22] type 00 class 0x0c0500
[    0.224235] pci 0000:00:1f.3: reg 0x10: [mem 0xc0619000-0xc06190ff 64bit]
[    0.224258] pci 0000:00:1f.3: reg 0x20: [io  0x3040-0x305f]
[    0.224504] pci 0000:01:00.0: [11ab:2b38] type 00 class 0x020000
[    0.224536] pci 0000:01:00.0: reg 0x10: [mem 0xc0500000-0xc05fffff 64bit pref]
[    0.224557] pci 0000:01:00.0: reg 0x18: [mem 0xc0400000-0xc04fffff 64bit pref]
[    0.224706] pci 0000:01:00.0: supports D1 D2
[    0.224710] pci 0000:01:00.0: PME# supported from D0 D1 D3hot D3cold
[    0.225071] pci 0000:00:1c.0: PCI bridge to [bus 01]
[    0.225080] pci 0000:00:1c.0:   bridge window [mem 0xc0400000-0xc05fffff]
[    0.227302] ACPI: PCI: Interrupt link LNKA configured for IRQ 0
[    0.227308] ACPI: PCI: Interrupt link LNKA disabled
[    0.227360] ACPI: PCI: Interrupt link LNKB configured for IRQ 0
[    0.227364] ACPI: PCI: Interrupt link LNKB disabled
[    0.227413] ACPI: PCI: Interrupt link LNKC configured for IRQ 0
[    0.227417] ACPI: PCI: Interrupt link LNKC disabled
[    0.227466] ACPI: PCI: Interrupt link LNKD configured for IRQ 0
[    0.227469] ACPI: PCI: Interrupt link LNKD disabled
[    0.227517] ACPI: PCI: Interrupt link LNKE configured for IRQ 0
[    0.227520] ACPI: PCI: Interrupt link LNKE disabled
[    0.227569] ACPI: PCI: Interrupt link LNKF configured for IRQ 0
[    0.227572] ACPI: PCI: Interrupt link LNKF disabled
[    0.227621] ACPI: PCI: Interrupt link LNKG configured for IRQ 0
[    0.227624] ACPI: PCI: Interrupt link LNKG disabled
[    0.227673] ACPI: PCI: Interrupt link LNKH configured for IRQ 0
[    0.227676] ACPI: PCI: Interrupt link LNKH disabled
[    0.227995] Low-power S0 idle used by default for system suspend
[    0.228496] acpi PNP0C0A:01: ACPI dock station (docks/bays count: 1)
[    0.229383] ACPI: EC: interrupt unblocked
[    0.229383] ACPI: EC: event unblocked
[    0.229383] ACPI: EC: EC_CMD/EC_SC=0x66, EC_DATA=0x62
[    0.229383] ACPI: EC: GPE=0x38
[    0.229383] ACPI: \_SB_.PCI0.SBRG.EC0_: Boot DSDT EC initialization complete
[    0.229383] ACPI: \_SB_.PCI0.SBRG.EC0_: EC: Used to handle transactions and events
[    0.232237] iommu: Default domain type: Translated 
[    0.232237] iommu: DMA domain TLB invalidation policy: lazy mode 
[    0.285069] pps_core: LinuxPPS API ver. 1 registered
[    0.285075] pps_core: Software ver. 5.3.6 - Copyright 2005-2007 Rodolfo Giometti <giometti@linux.it>
[    0.285084] PTP clock support registered
[    0.285096] EDAC MC: Ver: 3.0.0
[    0.285241] Registered efivars operations
[    0.285241] NetLabel: Initializing
[    0.285241] NetLabel:  domain hash size = 128
[    0.285241] NetLabel:  protocols = UNLABELED CIPSOv4 CALIPSO
[    0.285241] NetLabel:  unlabeled traffic allowed by default
[    0.285241] PCI: Using ACPI for IRQ routing
[    0.292582] PCI: pci_cache_line_size set to 64 bytes
[    0.292700] e820: reserve RAM buffer [mem 0x00058000-0x0005ffff]
[    0.292703] e820: reserve RAM buffer [mem 0x0009e000-0x0009ffff]
[    0.292704] e820: reserve RAM buffer [mem 0xa88ad000-0xabffffff]
[    0.292706] e820: reserve RAM buffer [mem 0xaa377000-0xabffffff]
[    0.292708] e820: reserve RAM buffer [mem 0x14fe00000-0x14fffffff]
[    0.292727] pci 0000:00:02.0: vgaarb: setting as boot VGA device
[    0.292727] pci 0000:00:02.0: vgaarb: bridge control possible
[    0.292727] pci 0000:00:02.0: vgaarb: VGA device added: decodes=io+mem,owns=io+mem,locks=none
[    0.292727] vgaarb: loaded
[    0.294228] clocksource: Switched to clocksource tsc-early
[    0.294398] VFS: Disk quotas dquot_6.6.0
[    0.294423] VFS: Dquot-cache hash table entries: 512 (order 0, 4096 bytes)
[    0.294522] pnp: PnP ACPI init
[    0.294866] system 00:00: [io  0x0680-0x069f] has been reserved
[    0.294877] system 00:00: [io  0xffff] has been reserved
[    0.294881] system 00:00: [io  0xffff] has been reserved
[    0.294885] system 00:00: [io  0xffff] has been reserved
[    0.294888] system 00:00: [io  0x1800-0x18fe] has been reserved
[    0.294892] system 00:00: [io  0x164e-0x164f] has been reserved
[    0.295013] system 00:02: [io  0x1854-0x1857] has been reserved
[    0.295800] system 00:03: [mem 0xfed1c000-0xfed1ffff] has been reserved
[    0.295808] system 00:03: [mem 0xfed10000-0xfed17fff] has been reserved
[    0.295812] system 00:03: [mem 0xfed18000-0xfed18fff] has been reserved
[    0.295817] system 00:03: [mem 0xfed19000-0xfed19fff] has been reserved
[    0.295821] system 00:03: [mem 0xe0000000-0xefffffff] has been reserved
[    0.295826] system 00:03: [mem 0xfed20000-0xfed3ffff] has been reserved
[    0.295830] system 00:03: [mem 0xfed90000-0xfed93fff] could not be reserved
[    0.295834] system 00:03: [mem 0xff000000-0xffffffff] has been reserved
[    0.295839] system 00:03: [mem 0xfee00000-0xfeefffff] has been reserved
[    0.295843] system 00:03: [mem 0xaf210000-0xaf21ffff] has been reserved
[    0.296206] system 00:04: [mem 0xfe104000-0xfe104fff] has been reserved
[    0.296212] system 00:04: [mem 0xfe106000-0xfe106fff] has been reserved
[    0.296916] pnp: PnP ACPI: found 5 devices
[    0.303298] clocksource: acpi_pm: mask: 0xffffff max_cycles: 0xffffff, max_idle_ns: 2085701024 ns
[    0.303391] NET: Registered PF_INET protocol family
[    0.303478] IP idents hash table entries: 65536 (order: 7, 524288 bytes, linear)
[    0.304908] tcp_listen_portaddr_hash hash table entries: 2048 (order: 3, 32768 bytes, linear)
[    0.304951] Table-perturb hash table entries: 65536 (order: 6, 262144 bytes, linear)
[    0.304996] TCP established hash table entries: 32768 (order: 6, 262144 bytes, linear)
[    0.305129] TCP bind hash table entries: 32768 (order: 8, 1048576 bytes, linear)
[    0.305362] TCP: Hash tables configured (established 32768 bind 32768)
[    0.305434] MPTCP token hash table entries: 4096 (order: 4, 98304 bytes, linear)
[    0.305474] UDP hash table entries: 2048 (order: 4, 65536 bytes, linear)
[    0.305494] UDP-Lite hash table entries: 2048 (order: 4, 65536 bytes, linear)
[    0.305545] NET: Registered PF_UNIX/PF_LOCAL protocol family
[    0.305556] NET: Registered PF_XDP protocol family
[    0.305571] pci 0000:00:1c.0: PCI bridge to [bus 01]
[    0.305587] pci 0000:00:1c.0:   bridge window [mem 0xc0400000-0xc05fffff]
[    0.305601] pci_bus 0000:00: resource 4 [io  0x0000-0x0cf7 window]
[    0.305606] pci_bus 0000:00: resource 5 [io  0x0d00-0xffff window]
[    0.305611] pci_bus 0000:00: resource 6 [mem 0x000a0000-0x000fffff window]
[    0.305615] pci_bus 0000:00: resource 7 [mem 0xfed40000-0xfed44fff window]
[    0.305620] pci_bus 0000:00: resource 8 [mem 0xaf200000-0xfeafffff window]
[    0.305625] pci_bus 0000:01: resource 1 [mem 0xc0400000-0xc05fffff]
[    0.306113] PCI: CLS 64 bytes, default 64
[    0.306135] DMAR: ACPI device "INT33C2:00" under DMAR at fed91000 as 00:15.1
[    0.306142] DMAR: ACPI device "INT33C3:00" under DMAR at fed91000 as 00:15.2
[    0.306152] DMAR: No ATSR found
[    0.306155] DMAR: No SATC found
[    0.306158] DMAR: dmar1: Using Queued invalidation
[    0.306190] Trying to unpack rootfs image as initramfs...
[    0.306222] i2c_designware INT33C2:00: Adding to iommu group 0
[    0.306235] i2c_designware INT33C3:00: Adding to iommu group 1
[    0.306409] pci 0000:00:00.0: Adding to iommu group 2
[    0.306424] pci 0000:00:03.0: Adding to iommu group 3
[    0.306436] pci 0000:00:14.0: Adding to iommu group 4
[    0.306452] pci 0000:00:16.0: Adding to iommu group 5
[    0.306465] pci 0000:00:1b.0: Adding to iommu group 6
[    0.306478] pci 0000:00:1c.0: Adding to iommu group 7
[    0.306496] pci 0000:00:1f.0: Adding to iommu group 8
[    0.306509] pci 0000:00:1f.2: Adding to iommu group 8
[    0.306520] pci 0000:00:1f.3: Adding to iommu group 8
[    0.306533] pci 0000:01:00.0: Adding to iommu group 9
[    0.307285] DMAR: ACPI name space devices didn't probe correctly
[    0.307291] DMAR: Intel(R) Virtualization Technology for Directed I/O
[    0.307294] PCI-DMA: Using software bounce buffering for IO (SWIOTLB)
[    0.307298] software IO TLB: mapped [mem 0x00000000a389d000-0x00000000a789d000] (64MB)
[    0.307836] Initialise system trusted keyrings
[    0.307851] Key type blacklist registered
[    0.307933] workingset: timestamp_bits=36 max_order=20 bucket_order=0
[    0.311298] zbud: loaded
[    0.311888] integrity: Platform Keyring initialized
[    0.311901] integrity: Machine keyring initialized
[    0.311908] Key type asymmetric registered
[    0.311914] Asymmetric key parser 'x509' registered
[    0.674601] Freeing initrd memory: 39224K
[    0.680969] alg: self-tests for CTR-KDF (hmac(sha256)) passed
[    0.681003] Block layer SCSI generic (bsg) driver version 0.4 loaded (major 248)
[    0.681050] io scheduler mq-deadline registered
[    0.682430] pcieport 0000:00:1c.0: PME: Signaling with IRQ 42
[    0.682554] shpchp: Standard Hot Plug PCI Controller Driver version: 0.4
[    0.686456] ACPI: thermal: [Firmware Bug]: Invalid critical threshold (98)
[    0.690039] thermal LNXTHERM:00: registered as thermal_zone0
[    0.690050] ACPI: thermal: Thermal Zone [TZ0] (-269 C)
[    0.690384] Serial: 8250/16550 driver, 4 ports, IRQ sharing enabled
[    0.691453] hpet_acpi_add: no address or irqs in _CRS
[    0.691478] Linux agpgart interface v0.103
[    0.696508] tpm_tis MSFT0101:00: 2.0 TPM (device-id 0x1A, rev-id 16)
[    0.722082] AMD-Vi: AMD IOMMUv2 functionality not available on this system - This is not a bug.
[    0.722557] i8042: PNP: No PS/2 controller found.
[    0.722562] i8042: Probing ports directly.
[    1.255665] i8042: Can't read CTR while initializing i8042
[    1.255672] i8042: probe of i8042 failed with error -5
[    1.255781] mousedev: PS/2 mouse device common for all mice
[    1.255820] rtc_cmos 00:01: RTC can wake from S4
[    1.256091] rtc_cmos 00:01: registered as rtc0
[    1.256132] rtc_cmos 00:01: setting system clock to 2024-10-28T19:18:31 UTC (1730143111)
[    1.256163] rtc_cmos 00:01: alarms up to one month, y3k, 242 bytes nvram
[    1.256363] intel_pstate: Intel P-state driver initializing
[    1.256688] ledtrig-cpu: registered to indicate activity on CPUs
[    1.256814] efifb: probing for efifb
[    1.256826] efifb: framebuffer at 0xb0000000, using 1920k, total 1920k
[    1.256830] efifb: mode is 800x600x32, linelength=3200, pages=1
[    1.256833] efifb: scrolling: redraw
[    1.256835] efifb: Truecolor: size=8:8:8:8, shift=24:16:8:0
[    1.257492] Console: switching to colour frame buffer device 100x37
[    1.258080] fb0: EFI VGA frame buffer device
[    1.267706] NET: Registered PF_INET6 protocol family
[    1.271710] Segment Routing with IPv6
[    1.271741] In-situ OAM (IOAM) with IPv6
[    1.271772] mip6: Mobile IPv6
[    1.271782] NET: Registered PF_PACKET protocol family
[    1.271838] mpls_gso: MPLS GSO support
[    1.272121] microcode: sig=0x40651, pf=0x40, revision=0x26
[    1.272152] microcode: Microcode Update Driver: v2.2.
[    1.272159] IPI shorthand broadcast: enabled
[    1.272206] sched_clock: Marking stable (1270187261, 1860816)->(1301273815, -29225738)
[    1.272443] registered taskstats version 1
[    1.272718] Loading compiled-in X.509 certificates
[    1.291029] Loaded X.509 cert 'Debian Secure Boot CA: 6ccece7e4c6c0d1f6149f3dd27dfcc5cbb419ea1'
[    1.291311] Loaded X.509 cert 'Debian Secure Boot Signer 2022 - linux: 14011249c2675ea8e5148542202005810584b25f'
[    1.292090] zswap: loaded using pool lzo/zbud
[    1.292482] Key type .fscrypt registered
[    1.292734] Key type fscrypt-provisioning registered
[    1.296877] Key type encrypted registered
[    1.297381] ima: Allocated hash algorithm: sha256
[    1.333927] tsc: Refined TSC clocksource calibration: 2494.225 MHz
[    1.334289] clocksource: tsc: mask: 0xffffffffffffffff max_cycles: 0x23f3ea95b09, max_idle_ns: 440795287034 ns
[    1.335022] clocksource: Switched to clocksource tsc
[    1.353180] ima: No architecture policies found
[    1.353578] evm: Initialising EVM extended attributes:
[    1.353940] evm: security.selinux
[    1.354277] evm: security.SMACK64 (disabled)
[    1.354597] evm: security.SMACK64EXEC (disabled)
[    1.354911] evm: security.SMACK64TRANSMUTE (disabled)
[    1.355154] evm: security.SMACK64MMAP (disabled)
[    1.355361] evm: security.apparmor
[    1.355564] evm: security.ima
[    1.355766] evm: security.capability
[    1.355967] evm: HMAC attrs: 0x1
[    1.451467] clk: Disabling unused clocks
[    1.452463] Freeing unused decrypted memory: 2036K
[    1.453043] Freeing unused kernel image (initmem) memory: 2796K
[    1.453239] Write protecting the kernel read-only data: 26624k
[    1.453928] Freeing unused kernel image (text/rodata gap) memory: 2040K
[    1.454418] Freeing unused kernel image (rodata/data gap) memory: 1164K
[    1.467028] x86/mm: Checked W+X mappings: passed, no W+X pages found.
[    1.467238] Run /init as init process
[    1.467435]   with arguments:
[    1.467436]     /init
[    1.467437]   with environment:
[    1.467438]     HOME=/
[    1.467439]     TERM=linux
[    1.467439]     BOOT_IMAGE=/boot/vmlinuz-6.1.0-26-amd64
[    1.467440]     selinux=0
[    1.544594] hid: raw HID events driver (C) Jiri Kosina
[    1.557883] dw_dmac INTL9C60:00: can't request region for resource [mem 0x00000000-0x00000fff]
[    1.558317] dw_dmac: probe of INTL9C60:00 failed with error -16
[    1.621066] ACPI: battery: Slot [BAT0] (battery present)
[    2.414520] hid-generic 0018:045E:07C4.0001: hidraw0: I2C HID v1.00 Device [MSHW0030:00 045E:07C4] on i2c-MSHW0030:00
[    2.415797] input: NTRG0001:01 1B96:1B05 Stylus as /devices/pci0000:00/INT33C3:00/i2c-1/i2c-NTRG0001:01/0018:1B96:1B05.0002/input/input0
[    2.416790] input: NTRG0001:01 1B96:1B05 Touchscreen as /devices/pci0000:00/INT33C3:00/i2c-1/i2c-NTRG0001:01/0018:1B96:1B05.0002/input/input1
[    2.418081] input: NTRG0001:01 1B96:1B05 Mouse as /devices/pci0000:00/INT33C3:00/i2c-1/i2c-NTRG0001:01/0018:1B96:1B05.0002/input/input2
[    2.419264] input: NTRG0001:01 1B96:1B05 as /devices/pci0000:00/INT33C3:00/i2c-1/i2c-NTRG0001:01/0018:1B96:1B05.0002/input/input3
[    2.419900] input: NTRG0001:01 1B96:1B05 as /devices/pci0000:00/INT33C3:00/i2c-1/i2c-NTRG0001:01/0018:1B96:1B05.0002/input/input4
[    2.421003] hid-generic 0018:1B96:1B05.0002: input,hidraw1: I2C HID v1.00 Mouse [NTRG0001:01 1B96:1B05] on i2c-NTRG0001:01
[    2.422533] i801_smbus 0000:00:1f.3: SPD Write Disable is set
[    2.423144] i801_smbus 0000:00:1f.3: SMBus using PCI interrupt
[    2.428706] SCSI subsystem initialized
[    2.429812] i2c i2c-2: 2/2 memory slots populated (from DMI)
[    2.438204] ACPI: bus type USB registered
[    2.438782] usbcore: registered new interface driver usbfs
[    2.439362] usbcore: registered new interface driver hub
[    2.439947] usbcore: registered new device driver usb
[    2.459298] libata version 3.00 loaded.
[    2.463720] xhci_hcd 0000:00:14.0: xHCI Host Controller
[    2.464321] xhci_hcd 0000:00:14.0: new USB bus registered, assigned bus number 1
[    2.466014] xhci_hcd 0000:00:14.0: hcc params 0x200077c1 hci version 0x100 quirks 0x000000000004b810
[    2.467322] xhci_hcd 0000:00:14.0: xHCI Host Controller
[    2.467891] xhci_hcd 0000:00:14.0: new USB bus registered, assigned bus number 2
[    2.468428] xhci_hcd 0000:00:14.0: Host supports USB 3.0 SuperSpeed
[    2.468997] usb usb1: New USB device found, idVendor=1d6b, idProduct=0002, bcdDevice= 6.01
[    2.469150] ahci 0000:00:1f.2: version 3.0
[    2.469521] usb usb1: New USB device strings: Mfr=3, Product=2, SerialNumber=1
[    2.469914] usb usb1: Product: xHCI Host Controller
[    2.470225] usb usb1: Manufacturer: Linux 6.1.0-26-amd64 xhci-hcd
[    2.470521] usb usb1: SerialNumber: 0000:00:14.0
[    2.470978] hub 1-0:1.0: USB hub found
[    2.471281] hub 1-0:1.0: 9 ports detected
[    2.471908] usb usb2: New USB device found, idVendor=1d6b, idProduct=0003, bcdDevice= 6.01
[    2.472204] usb usb2: New USB device strings: Mfr=3, Product=2, SerialNumber=1
[    2.472479] usb usb2: Product: xHCI Host Controller
[    2.472748] usb usb2: Manufacturer: Linux 6.1.0-26-amd64 xhci-hcd
[    2.473012] usb usb2: SerialNumber: 0000:00:14.0
[    2.473389] hub 2-0:1.0: USB hub found
[    2.473668] hub 2-0:1.0: 4 ports detected
[    2.474042] usb: port power management may be unreliable
[    2.479747] ahci 0000:00:1f.2: AHCI 0001.0300 32 slots 3 ports 6 Gbps 0x1 impl SATA mode
[    2.480138] ahci 0000:00:1f.2: flags: 64bit ncq pm led clo only pio slum part deso sadm sds apst 
[    2.480917] scsi host0: ahci
[    2.481540] scsi host1: ahci
[    2.482038] scsi host2: ahci
[    2.482426] ata1: SATA max UDMA/133 abar m2048@0xc061a000 port 0xc061a100 irq 44
[    2.482711] ata2: DUMMY
[    2.482978] ata3: DUMMY
[    2.493485] input: NTRG0001:01 1B96:1B05 Stylus as /devices/pci0000:00/INT33C3:00/i2c-1/i2c-NTRG0001:01/0018:1B96:1B05.0002/input/input5
[    2.494389] input: NTRG0001:01 1B96:1B05 as /devices/pci0000:00/INT33C3:00/i2c-1/i2c-NTRG0001:01/0018:1B96:1B05.0002/input/input6
[    2.495253] hid-multitouch 0018:1B96:1B05.0002: input,hidraw0: I2C HID v1.00 Mouse [NTRG0001:01 1B96:1B05] on i2c-NTRG0001:01
[    2.729934] usb 1-7: new high-speed USB device number 2 using xhci_hcd
[    2.797144] ata1: SATA link up 6.0 Gbps (SStatus 133 SControl 300)
[    2.799620] ata1.00: ATA-9: SAMSUNG MZMTE128HMGR-000MV, EXT41M0Q, max UDMA/133
[    2.800481] ata1.00: NCQ Send/Recv Log not supported
[    2.800905] ata1.00: 250069680 sectors, multi 16: LBA48 NCQ (depth 32), AA
[    2.801762] ata1.00: Features: Dev-Sleep
[    2.803207] ata1.00: NCQ Send/Recv Log not supported
[    2.804324] ata1.00: configured for UDMA/133
[    2.805057] scsi 0:0:0:0: Direct-Access     ATA      SAMSUNG MZMTE128 1M0Q PQ: 0 ANSI: 5
[    2.819323] sd 0:0:0:0: [sda] 250069680 512-byte logical blocks: (128 GB/119 GiB)
[    2.819646] sd 0:0:0:0: [sda] Write Protect is off
[    2.819934] sd 0:0:0:0: [sda] Mode Sense: 00 3a 00 00
[    2.819964] sd 0:0:0:0: [sda] Write cache: enabled, read cache: enabled, doesn't support DPO or FUA
[    2.820572] sd 0:0:0:0: [sda] Preferred minimum I/O size 512 bytes
[    2.823066]  sda: sda1 sda2 sda3
[    2.823632] sd 0:0:0:0: [sda] Attached SCSI disk
[    2.887891] usb 1-7: New USB device found, idVendor=045e, idProduct=07be, bcdDevice=21.52
[    2.888350] usb 1-7: New USB device strings: Mfr=1, Product=2, SerialNumber=0
[    2.888762] usb 1-7: Product: Microsoft LifeCam Front
[    2.889116] usb 1-7: Manufacturer: QCM
[    2.945496] PM: Image not found (code -22)
[    3.021891] usb 1-8: new high-speed USB device number 3 using xhci_hcd
[    3.047398] EXT4-fs (sda2): mounted filesystem with ordered data mode. Quota mode: none.
[    3.085842] Not activating Mandatory Access Control as /sbin/tomoyo-init does not exist.
[    3.175332] usb 1-8: New USB device found, idVendor=045e, idProduct=07bf, bcdDevice=21.52
[    3.175801] usb 1-8: New USB device strings: Mfr=1, Product=2, SerialNumber=0
[    3.176244] usb 1-8: Product: Microsoft LifeCam Rear
[    3.176690] usb 1-8: Manufacturer: QCM
[    3.483494] device-mapper: core: CONFIG_IMA_DISABLE_HTABLE is disabled. Duplicate IMA measurements will not be recorded in the IMA log.
[    3.485309] device-mapper: uevent: version 1.0.3
[    3.487074] device-mapper: ioctl: 4.47.0-ioctl (2022-07-28) initialised: dm-devel@redhat.com
[    3.487212] fuse: init (API version 7.38)
[    3.491861] pstore: Using crash dump compression: deflate
[    3.494687] loop: module loaded
[    3.500437] pstore: Registered efi as persistent store backend
[    3.517160] EXT4-fs (sda2): re-mounted. Quota mode: none.
[    3.553421] ACPI: bus type drm_connector registered
[    3.786675] sd 0:0:0:0: Attached scsi generic sg0 type 0
[    3.841859] ACPI: AC: AC Adapter [AC0] (on-line)
[    3.874351] input: Surface Pro 3/4 Buttons as /devices/LNXSYSTM:00/LNXSYBUS:00/PNP0A08:00/device:14/MSHW0028:00/input/input11
[    3.893888] input: Lid Switch as /devices/LNXSYSTM:00/LNXSYBUS:00/PNP0A08:00/device:14/PNP0C0D:00/input/input10
[    3.902147] surface_pro3_button MSHW0028:00: Surface Pro 3/4 Buttons [VGBI]
[    3.908720] ACPI: button: Lid Switch [LID0]
[    3.925271] input: PC Speaker as /devices/platform/pcspkr/input/input12
[    3.938855] iTCO_vendor_support: vendor-support=0
[    3.948657] mc: Linux media interface: v0.10
[    3.994837] videodev: Linux video capture interface: v2.00
[    4.043016] Adding 1000444k swap on /dev/sda3.  Priority:-2 extents:1 across:1000444k SSFS
[    4.051304] iTCO_wdt iTCO_wdt.1.auto: Found a Lynx Point_LP TCO device (Version=2, TCOBASE=0x1860)
[    4.053389] iTCO_wdt iTCO_wdt.1.auto: initialized. heartbeat=30 sec (nowayout=0)
[    4.054221] surface_gpe: no compatible Microsoft Surface device found, exiting
[    4.058152] cfg80211: Loading compiled-in X.509 certificates for regulatory database
[    4.059159] cfg80211: Loaded X.509 cert 'benh@debian.org: 577e021cb980e0e820821ba7b54b4961b8b4fadf'
[    4.060823] cfg80211: Loaded X.509 cert 'romain.perier@gmail.com: 3abbc6ec146e09d1b6016ab9d6cf71dd233f0328'
[    4.062506] cfg80211: Loaded X.509 cert 'sforshee: 00b28ddf47aef9cea7'
[    4.063491] cfg80211: Loaded X.509 cert 'wens: 61c038651aabdcf94bd0ac7ff06c7248db18c600'
[    4.065493] platform regulatory.0: firmware: direct-loading firmware regulatory.db
[    4.066546] platform regulatory.0: firmware: direct-loading firmware regulatory.db.p7s
[    4.075535] usb 1-7: Found UVC 1.50 device Microsoft LifeCam Front (045e:07be)
[    4.090795] input: Microsoft LifeCam Front: Micros as /devices/pci0000:00/0000:00:14.0/usb1/1-7/1-7:1.0/input/input13
[    4.092993] RAPL PMU: API unit is 2^-32 Joules, 4 fixed counters, 655360 ms ovfl timer
[    4.093654] RAPL PMU: hw unit of domain pp0-core 2^-14 Joules
[    4.094375] RAPL PMU: hw unit of domain package 2^-14 Joules
[    4.095086] RAPL PMU: hw unit of domain dram 2^-14 Joules
[    4.095704] RAPL PMU: hw unit of domain pp1-gpu 2^-14 Joules
[    4.099022] snd_hda_intel 0000:00:03.0: enabling device (0000 -> 0002)
[    4.100426] snd_hda_intel 0000:00:1b.0: enabling device (0000 -> 0002)
[    4.103503] usb 1-8: Found UVC 1.50 device Microsoft LifeCam Rear (045e:07bf)
[    4.107954] input: Microsoft LifeCam Rear: Microso as /devices/pci0000:00/0000:00:14.0/usb1/1-8/1-8:1.0/input/input14
[    4.109565] usbcore: registered new interface driver uvcvideo
[    4.165309] mwifiex_pcie 0000:01:00.0: no quirks enabled
[    4.166053] mwifiex_pcie 0000:01:00.0: enabling device (0000 -> 0002)
[    4.166960] mwifiex_pcie: PCI memory map Virt0: 00000000a2694baa PCI memory map Virt2: 00000000f7bc3eb8
[    4.168435] cryptd: max_cpu_qlen set to 1000
[    4.177268] mwifiex_pcie 0000:01:00.0: firmware: direct-loading firmware mrvl/pcie8897_uapsta.bin
[    4.218399] AVX2 version of gcm_enc/dec engaged.
[    4.219067] AES CTR mode by8 optimization enabled
[    4.226721] snd_hda_codec_realtek hdaudioC1D0: autoconfig for ALC288: line_outs=1 (0x14/0x0/0x0/0x0/0x0) type:speaker
[    4.227976] snd_hda_codec_realtek hdaudioC1D0:    speaker_outs=0 (0x0/0x0/0x0/0x0/0x0)
[    4.228607] snd_hda_codec_realtek hdaudioC1D0:    hp_outs=1 (0x21/0x0/0x0/0x0/0x0)
[    4.229229] snd_hda_codec_realtek hdaudioC1D0:    mono: mono_out=0x0
[    4.229830] snd_hda_codec_realtek hdaudioC1D0:    inputs:
[    4.231492] snd_hda_codec_realtek hdaudioC1D0:      Mic=0x18
[    4.232070] snd_hda_codec_realtek hdaudioC1D0:      Internal Mic=0x12
[    4.519182] input: HDA Digital PCBeep as /devices/pci0000:00/0000:00:1b.0/sound/card1/input15
[    4.521143] input: HDA Intel PCH Mic as /devices/pci0000:00/0000:00:1b.0/sound/card1/input16
[    4.526029] input: HDA Intel PCH Headphone as /devices/pci0000:00/0000:00:1b.0/sound/card1/input17
[    4.528276] i915 0000:00:02.0: can't derive routing for PCI INT A
[    4.528873] i915 0000:00:02.0: PCI INT A: not connected
[    4.532328] Console: switching to colour dummy device 80x25
[    4.532761] i915 0000:00:02.0: vgaarb: deactivate vga console
[    4.536386] i915 0000:00:02.0: vgaarb: changed VGA decodes: olddecodes=io+mem,decodes=io+mem:owns=io+mem
[    4.638317] [drm] Initialized i915 1.6.0 20201103 for 0000:00:02.0 on minor 0
[    4.644101] ACPI: video: Video Device [GFX0] (multi-head: yes  rom: no  post: no)
[    4.644373] input: Video Bus as /devices/LNXSYSTM:00/LNXSYBUS:00/PNP0A08:00/LNXVIDEO:00/input/input18
[    4.644511] snd_hda_intel 0000:00:03.0: bound 0000:00:02.0 (ops i915_audio_component_bind_ops [i915])
[    4.659610] fbcon: i915drmfb (fb0) is primary device
[    5.577402] mwifiex_pcie 0000:01:00.0: info: FW download over, size 723540 bytes
[    5.808714] Console: switching to colour frame buffer device 135x45
[    5.827265] i915 0000:00:02.0: [drm] fb0: i915drmfb frame buffer device
[    5.884876] input: HDA Intel HDMI HDMI/DP,pcm=3 as /devices/pci0000:00/0000:00:03.0/sound/card0/input19
[    5.884941] input: HDA Intel HDMI HDMI/DP,pcm=7 as /devices/pci0000:00/0000:00:03.0/sound/card0/input20
[    5.885152] input: HDA Intel HDMI HDMI/DP,pcm=8 as /devices/pci0000:00/0000:00:03.0/sound/card0/input21
[    5.904343] intel_rapl_common: Found RAPL domain package
[    5.904368] intel_rapl_common: Found RAPL domain core
[    5.904384] intel_rapl_common: Found RAPL domain uncore
[    5.904402] intel_rapl_common: Found RAPL domain dram
[    5.904429] intel_rapl_common: RAPL package-0 domain dram locked by BIOS
[    6.076649] NET: Registered PF_QIPCRTR protocol family
[    6.321891] usb 1-6: new high-speed USB device number 4 using xhci_hcd
[    6.474826] usb 1-6: New USB device found, idVendor=1286, idProduct=204b, bcdDevice=32.01
[    6.474835] usb 1-6: New USB device strings: Mfr=1, Product=2, SerialNumber=3
[    6.474838] usb 1-6: Product: Bluetooth and Wireless LAN Composite Device
[    6.474841] usb 1-6: Manufacturer: Marvell
[    6.474843] usb 1-6: SerialNumber: 0000000000000000
[    6.496010] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)
[    6.549980] mwifiex_pcie 0000:01:00.0: WLAN FW is active
[    6.612574] Bluetooth: Core ver 2.22
[    6.612621] NET: Registered PF_BLUETOOTH protocol family
[    6.612625] Bluetooth: HCI device and connection manager initialized
[    6.612630] Bluetooth: HCI socket layer initialized
[    6.612634] Bluetooth: L2CAP socket layer initialized
[    6.612642] Bluetooth: SCO socket layer initialized
[    6.627911] usbcore: registered new interface driver btusb
[    6.629335] Bluetooth: hci0: unexpected event for opcode 0x0000
[    6.629814] Bluetooth: hci0: unexpected event for opcode 0x0000
[    6.646409] mwifiex_pcie 0000:01:00.0: info: MWIFIEX VERSION: mwifiex 1.0 (15.68.19.p21) 
[    6.646417] mwifiex_pcie 0000:01:00.0: driver_version = mwifiex 1.0 (15.68.19.p21) 
[    6.673938] mwifiex_pcie 0000:01:00.0 wlp1s0: renamed from mlan0
[    6.728272] Bluetooth: BNEP (Ethernet Emulation) ver 1.3
[    6.728279] Bluetooth: BNEP filters: protocol multicast
[    6.728284] Bluetooth: BNEP socket layer initialized
[    6.775554] Bluetooth: MGMT ver 1.22
[    6.781061] NET: Registered PF_ALG protocol family
[    9.878767] mwifiex_pcie 0000:01:00.0: info: trying to associate to bssid cc:2d:e0:a3:ae:92
[    9.981834] mwifiex_pcie 0000:01:00.0: info: associated to bssid cc:2d:e0:a3:ae:92 successfully
[    9.989178] IPv6: ADDRCONF(NETDEV_CHANGE): wlp1s0: link becomes ready
[   10.126637] mwifiex_pcie 0000:01:00.0: wrong rx packet offset: len=106, rx_pkt_off=90
```

</details>

**Comment by dynek** [comment]
Oh well, sometimes posting a question helps resolve it for some reasons 🤷🏻‍♂️
Disabling power management on wlp1s0 fixed the latency/slowness issue, see https://unix.stackexchange.com/questions/269661/how-to-turn-off-wireless-power-management-permanently

But the domes flood `mwifiex_pcie 0000:01:00.0: wrong rx packet offset: len=106, rx_pkt_off=90` is still there. Don't know if I should try to fix this...
